### ISSUE TITLE
Newsletter | Versandtoptionen dauerhaft vorbelegen

### ISSUE THREAD

**Issue by sascha-mueller** [contao/core#8086]
Direkt vor dem Versenden des Newsletters kann ich folgende Felder ändern:

* Mails per cycle
* Timeout in seconds
* Offset

Leider muss man das bei jedem Versandt tun, da die Werte nicht gespeichert werden und auch nicht an zentraler Stelle verwaltet werden können.

Leider ist dies Ungünstig, wenn der User nicht so erfahren ist, extrem selten oder extrem viele Newsletter verfasst.

Ich würde mir daher für meine Redakteure wünschen, dass diese Felder zumindest an einer Stelle zentral (zb localconfig.php) vorbelegt werden können.

**Comment by sascha-mueller** [comment]
Wie krieg ich das denn noch für die 3.5 hingebogen?
Möchte da nicht weitere 3,5 Jahre drauf verzichten :-)

**Comment by leofeyer** [comment]
Wieso 3,5 Jahre? Die 4.2 kommt in 6 Monaten.

**Comment by sascha-mueller** [comment]
Weil es bisher immer gute Gründe gab bei der LTS zu bleiben.
Siehst Du ne Möglichkeit dass ich das mit etwas Hilfe für mich anpassen kann oder ist das ein zu starker Eingriff in den Core und deshalb nicht zu trennen?

**Comment by leofeyer** [comment]
Nein, das lässt sich schon machen. Du musst nur die fest eingestellten Value-Attribute in der Datei [Newsletter.php](https://github.com/contao/core/blob/master/system/modules/newsletter/classes/Newsletter.php#L262) durch `'.\Config::get('soWieDuDeineOptionenGerneNennst').'` ersetzen.

**Comment by leofeyer** [comment]
Da die Änderungen sowieso durch das Live-Update potentiell überschrieben werden, kannst Du die Attribute auch direkt setzen. Ohne Umweg über die `localconfig.php` meine ich.

**Comment by sascha-mueller** [comment]
Danke. Damit komme ich erstmal ne Weile zureckt, auch wenn es nicht Updatesicher ist :-)
